config/initializers/strong_migrations.rb

```diff
@@ -1,26 +1,30 @@
-# Mark existing migrations as safe
-StrongMigrations.start_after = 20220604103854
+# frozen_string_literal: true
 
-# Set timeouts for migrations
-# If you use PgBouncer in transaction mode, delete these lines and set timeouts on the database user
-StrongMigrations.lock_timeout = 10.seconds
-StrongMigrations.statement_timeout = 1.hour
+if Rails.env.development?
+  # Mark existing migrations as safe
+  StrongMigrations.start_after = '20220604103854'
```

Comment:
here I endend up leaving it as an int and asking rubocop to ignore it, after I got an error. that solved it

Reply:
In the feature, this should be reason enough for you to request changes in the PR, since this would probably break migrations :x